docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and, de-facto, the members of the Coordination Committee.
+
+Any contribution on these topics renders the user as an official Gammapy contributor.
```

Comment:
Any contribution, really? 
I would not keep the sentence.

Reply:
Either you fix a threshold or not. Indeed, this sentence means 'no threshold'. This is why it is useful, you get it.
If you propose thresholds, can you make a proposal?

Reply:
Yes, defining a threshold is tricky to impossible. I think the only practical threshold is >0.